xml/deployment_rgw.xml

```diff
@@ -288,9 +288,10 @@ sudo mv ceph.client.rgw.keyring /etc/ceph/ceph.client.rgw.keyring</screen>
        <para>
         Copy the &rgw; keyring to the &rgw; host:
        </para>
-<screen>sudo scp \
+<screen>&prompt.cephuser;sudo scp \
 /var/lib/ceph/radosgw/ceph-<replaceable>rgw_name</replaceable>/keyring \
 <replaceable>rgw_host</replaceable>:/var/lib/ceph/radosgw/ceph-<replaceable>rgw_name</replaceable>/keyring</screen>
+      <remark role="fixme">Does the scp command fail because it is not executed as remote root but local root?</remark>
```

Comment:
should be fine, remark can be removed. When scp is executed as root, remote user is root as well.